gc/buildreaper_test.go

```diff
@@ -32,6 +32,7 @@ var _ = Describe("BuildReaper", func() {
 			buildReaperLogger,
 			fakePipelineFactory,
 			batchSize,
+			NewBuildLogRetentionCalculator(0, 0),
```

Comment:
Hmm I'm a little worried that the coverage here isn't sufficient. The build reaper could have been unchanged, either in the line checking if it's 0 or in the line adding it to `db.Page`, and all the tests would have passed.

Could tests be added here, either instead of or in addition to the tests for the calculator? (Depending on where the tests feel more natural to write, either with fakes or with a real object.)